bundles/org.openhab.binding.ipcamera/README.md

```diff
@@ -13,7 +13,7 @@ To see what features each brand has implemented from their APIs, please see this
 - Check this readme for any setup steps for your brand.
 - Check if the camera is offline, if so there will be a reason listed.
 - Always look at the log files with TRACE enabled, as any FFmpeg and camera errors may not reach the INFO logs.
-To enable TRACE logging, enter this in the openHAB console `log:set TRACE org.openhab.binding.ipcamera`.
+To enable TRACE logging, enter this in the openHAB console `log:set TRACE org.openhab.binding.ipcamera`, or with the log viewer add `org.openhab.binding.ipcamera` and use the drop down list to select TRACE.
```

Comment:
Similar as the other comment. Please point to the logging documentation

Reply:
The logging documentation does cover the new Log Viewer feature, nor how to use the other methods other then the console. I will remove the information completely.